packages/client/components/PokerEstimateHeaderCard.tsx

```diff
@@ -4,12 +4,58 @@ import {createFragmentContainer} from 'react-relay'
 import {PokerEstimateHeaderCard_stage} from '../__generated__/PokerEstimateHeaderCard_stage.graphql'
 import PokerEstimateHeaderCardContent from './PokerEstimateHeaderCardContent'
 import PokerEstimateHeaderCardError from './PokerEstimateHeaderCardError'
-import PokerEstimateHeaderCardGitHub from './PokerEstimateHeaderCardGitHub'
 import PokerEstimateHeaderCardParabol from './PokerEstimateHeaderCardParabol'
 
 interface Props {
   stage: PokerEstimateHeaderCard_stage
 }
+
+type Integration = NonNullable<NonNullable<PokerEstimateHeaderCard_stage['task']>['integration']>
```

Comment:
+1 The outer `NunNullable<>` is not necessary since you use the type later with `Integration | null`. For me it also doesn't add to the readability since I rely on hover types to know what's the final type.